Extension/src/LanguageServer/ui.ts

```diff
@@ -296,6 +309,21 @@ export class OldUI implements UI {
         }
     }
 
+    private compilerTimout?: NodeJS.Timeout;
+    public async showCompilerStatusIcon(show: boolean): Promise<void> {
+        if (this.compilerTimout) {
+            clearTimeout(this.compilerTimout);
+        }
+        const action: string = "";
+        if (show) {
+            this.compilerTimout = setTimeout(() => { this.compilerStatusItem.show(); }, 15000);
```

Comment:
I think there was a condition for doing a timeout vs. just showing it.  Check with @sean-mcmanus on that.

Reply:
@sean-mcmanus what do you think?

Reply:
what?

Reply:
I think you mentioned that you wanted to force the status bar to be shown before the timeout if the user explicitly closes the prompts. I personally didn't think it mattered too much though. Up to you guys.